service/dap/server.go

```diff
@@ -2670,6 +2678,11 @@ func (s *Server) resetHandlesForStoppedEvent() {
 	s.exceptionErr = nil
 }
 
+func processExited(stopReason proc.StopReason, state *api.DebuggerState, err error) bool {
```

Comment:
The stopReason parameter should be removed.

Reply:
Done